packages/shared/src/components/SearchEmptyScreen.tsx

```diff
@@ -1,19 +1,16 @@
 import React, { ReactElement } from 'react';
 import { SearchIcon as MagnifyingIcon } from './icons';
-import { EmptyScreenIcon } from './EmptyScreen';
+import { IconSize } from './Icon';
 
 export default function SearchEmptyScreen(): ReactElement {
   return (
-    <div
-      className="flex w-full flex-col self-center px-6"
-      style={{ maxWidth: '32.5rem' }}
-    >
+    <div className="flex w-full max-w-[32rem] flex-col items-center gap-4 self-center px-6">
       <MagnifyingIcon
-        className="self-center text-theme-label-disabled"
-        style={EmptyScreenIcon.style}
+        className="text-overlay-tertiary-salt"
```

Comment:
Was it not text-theme-label-disabled?
It seems it's named that way in figma (maybe we have a mismatch here naming wise, or they changed it)

Reply:
Basically they are both `salt.90` with `32%` opacity, I'll update with disabled. Just the matter of colors being confusing to find I guess. Good catch 👍

Reply:
Updated 3c2008e164e663dad4bf68ad28646486eb8965b2